{"text": "2025-04-16 12:19:09 | INFO | Rason:main:277 - Initializing agent...\n", "record": {"elapsed": {"repr": "0:00:00.687487", "seconds": 0.687487}, "exception": null, "extra": {}, "file": {"name": "Rason.py", "path": "/Users/sin/ai/ai-wallet/app/test/thinkingagent/Rason.py"}, "function": "main", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 277, "message": "Initializing agent...", "module": "Rason", "name": "__main__", "process": {"id": 80813, "name": "MainProcess"}, "thread": {"id": 8331790400, "name": "MainThread"}, "time": {"repr": "2025-04-16 12:19:09.966032+08:00", "timestamp": 1744777149.966032}}}
{"text": "2025-04-16 12:19:10 | CRITICAL | Rason:main:300 - Main execution failed: RetryError[<Future at 0x10a66b750 state=finished raised TypeError>]\n", "record": {"elapsed": {"repr": "0:00:00.767727", "seconds": 0.767727}, "exception": null, "extra": {}, "file": {"name": "Rason.py", "path": "/Users/sin/ai/ai-wallet/app/test/thinkingagent/Rason.py"}, "function": "main", "level": {"icon": "☠️", "name": "CRITICAL", "no": 50}, "line": 300, "message": "Main execution failed: RetryError[<Future at 0x10a66b750 state=finished raised TypeError>]", "module": "Rason", "name": "__main__", "process": {"id": 80813, "name": "MainProcess"}, "thread": {"id": 8331790400, "name": "MainThread"}, "time": {"repr": "2025-04-16 12:19:10.046272+08:00", "timestamp": 1744777150.046272}}}
